apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -372,19 +372,22 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             inboundProtocolConfig.issuer = samlConfigurations.issuer;
         }
 
-        const extensions: ResourceTabPaneInterface[] = applicationConfig.editApplication.getTabExtensions({
-            application: application,
-            content: template.content.quickStart,
-            inboundProtocolConfig: inboundProtocolConfig,
-            inboundProtocols: inboundProtocolList,
-            onApplicationUpdate: () => {
-                onUpdate(application?.id);
-            },
-            onTriggerTabUpdate: (tabIndex: number) => {
-                setActiveTabIndex(tabIndex);
+        const extensions: ResourceTabPaneInterface[] = applicationConfig.editApplication.getTabExtensions(
```

Comment:
Can we improve formatting here.
```suggestion
        const extensions: ResourceTabPaneInterface[] = applicationConfig.editApplication.getTabExtensions({
```

Reply:
Added the `{` to next line to identify the two arguments clearly since I added a new arguement to the function.